shaq_overflow/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+
+  def create
+    comment = Comment.new(comments_params)
+    comment.commentable_id = params[:question_id]
+    comment.user_id = current_user.id
```

Comment:
Is this guaranteed to be a comment on a question rather than anything else? where would comments on answers map to?